src/components/InlineErrorText.js

```diff
@@ -19,7 +21,7 @@ const InlineErrorText = (props) => {
     }
 
     return (
-        <Text style={[styles.formError, styles.mt1]}>{props.children}</Text>
+        <Text style={[styles.formError, styles.mt1, ...props.style]}>{props.children}</Text>
```

Comment:
I created a style prop to `InlineErrorText` so it can customize its styles only on places where the component is mounted, other `InlineErrorText` components can have style prop or none of it and it wont affect others. What do you think?